package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "searchRelevance",
+  "version": "2.3.0.0",
```

Comment:
Please update to 3.0.0.0 and then 2.4.0.0 when we cut the 2.x branch.

Reply:
@mingkunm Can you please update the name here to "search-relevance-dashboards"